.github/pull_request_template.md

```diff
@@ -1,19 +1,15 @@
-# Grant Application
+### Project Abstract
 
-## Abstract
-
-> Provide a brief description of your project here summarising key points (1-2 paragraphs).
->
-> If your application is a follow-up to a previous one, please mention which one in the first line of the abstract and include a link to previous pull requests if applicable.
+> Please provide a brief description of your project here summarising key points (1-2 paragraphs).
 >
-> The details should be in the application document that is being added to the repository via this pull request.
+> If your application is a follow-up to a previous grant, please mention which one in the first line of the abstract and include a link to previous pull requests if applicable.
 
-## Checklist
+### Application Checklist
 
-- [ ] The [application template](https://github.com/w3f/Grants-Program/blob/master/applications/application-template.md) has been copied, renamed ( `project_name.md`) and updated.
+- [x] The [application template](https://github.com/w3f/Grants-Program/blob/master/applications/application-template.md) has been copied, renamed ( `project_name.md`) and updated.
```

Comment:
Shouldn't be pre-ticked, no?

Reply:
Seems not everybody knows how to tick these boxes, this is meant as an example. Following https://github.com/w3f/Grant-Milestone-Delivery/blob/master/.github/pull_request_template.md

Reply:
How about:

```
- [x] I have ticked this box
```